### ISSUE TITLE
[Effect Suggestion] My Super Cool Effect Idea

### ISSUE THREAD

**Issue by SomeRandomExplorer** [gta-chaos-mod/ChaosModV#1617]
**Name of effect**
CSS is Required

**Effect Description**
Replace all map textures with a pink and black checkered texture

**Comment by DEATHCON1** [comment]
In this game, the missing texture texture is a black and white checkered print. But I have only ever seen it when another online player glitches out their outfit.

**Comment by SomeRandomExplorer** [comment]
> In this game, the missing texture texture is a black and white checkered print. But I have only ever seen it when another online player glitches out their outfit.

It is a reference to Garry’s Mod and how some maps require Counter-Strike: Source in order to work properly, hence the choice of black and pink checkered textures

**Comment by DEATHCON1** [comment]
Yes, but you can't make it pink and black in this game, I don't think it's possible. It will have to be black and white.

**Comment by SomeRandomExplorer** [comment]
> Yes, but you can't make it pink and black in this game, I don't think it's possible. It will have to be black and white.

Sad
